forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/DefaultTokenManagerTest.java

```diff
@@ -7,9 +7,14 @@
 
 package org.forgerock.android.auth;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
Update copyright

Reply:
Reverted the changes and left the original copyright note...